pages/index.html

```diff
@@ -10,8 +10,8 @@
   <h1 id=logo><span>LiveWires</span></h1>
   <div class="row row-in-hero col-paragraph">
     <h2 class=tagline>Technical activity week for 12–15s</h2>
-    <p class=tagline>7th–14th August 2022 • Chafyn Grove School, Salisbury</p>
-    <a class="cta" href="#book">Bookings now open</a>
+    <p class=tagline>August 2023 • Chafyn Grove School, Salisbury</p>
+    <a class="cta" href="/prereg">Pre-register now!</a>
```

Comment:
The link at the bottom of the page open in a new tab; I think that we should have consistent behaviour here too please